### PR TITLE
Add ethereum & scw reaction

### PR DESCRIPTION
**Pull request by 0xpanoramix**
Signed-off-by: Luca Georges Francois <luca.georges-francois@epitech.eu>

# Description

This PR adds the ethereum reaction to send a transaction on the Ropsten testnet.
It updates the protobuffers definition to drop the microsoft service as this company is shit.

# Changes include

- [ ] Bugfix (non-breaking change that solves an issue)
- [X] New feature (non-breaking change that adds functionality)
- [ ] Breaking change (change that is not backwards-compatible and/or changes current functionality)

# Checklist

- [X] I have assigned this PR to myself
- [X] I have added at least 1 reviewer
- [X] I have tested this code
- [X] I have added sufficient documentation in the code

### CODE REVIEW

File: obelix/panoramix/services/ethereum/client.go [hunk 1]
```diff
@@ -0,0 +1,127 @@
...
+}
+
+func NewClient(ctx context.Context) *Client {
+	return &Client{
+		ctx: ctx,
+	}
+}
+
+func (c *Client) configure(owner string) error {
+	clt, err := ethclient.Dial(os.Getenv("INFURA_ENDPOINT"))
```
Comment (Cleopha) [inline obelix/panoramix/services/ethereum/client.go]: pls in init and check if INFURA_ENDPOINT is set

File: obelix/panoramix/services/ethereum/client.go [hunk 2]
```diff
@@ -0,0 +1,127 @@
...
+	if err != nil {
+		return fmt.Errorf("failed to create ethereum client: %w", err)
+	}
+
+	c.clt = clt
+
+	credentialsClient, err := credentials.NewClient(os.Getenv("CREDENTIAL_API_PORT"))
```
Comment (Cleopha) [inline obelix/panoramix/services/ethereum/client.go]: pls in init and check if CREDENTIAL_API_PORT is set

**Review by Cleopha — CHANGES_REQUESTED** [review]
LGTM

File: obelix/panoramix/services/ethereum/client.go (hunk 1 above)
Comment (0xpanoramix) [reply obelix/panoramix/services/ethereum/client.go]: Fixed in https://github.com/Cleopha/ifttt-like/pull/76/commits/485f0b1571ae6d34ece509592cc9a4cde32bc17b

File: obelix/panoramix/services/ethereum/client.go (hunk 2 above)
Comment (0xpanoramix) [reply obelix/panoramix/services/ethereum/client.go]: Fixed in https://github.com/Cleopha/ifttt-like/pull/76/commits/485f0b1571ae6d34ece509592cc9a4cde32bc17b

**Review by Cleopha — APPROVED** [review]

**Review by TomChv — APPROVED** [review]
LGTM

**Pull request merged**